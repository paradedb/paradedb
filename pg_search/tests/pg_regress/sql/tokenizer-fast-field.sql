DROP TABLE IF EXISTS tokenizer_fast;
CREATE TABLE tokenizer_fast (
    id serial8 not null primary key,
    t text
);

INSERT INTO tokenizer_fast (t) VALUES ('This is a TEST');

CREATE INDEX idxtokenizer_fast ON tokenizer_fast USING bm25 (id, (t::pdb.literal)) WITH (key_field = 'id');

SELECT * FROM paradedb.schema('idxtokenizer_fast') ORDER BY name;
SELECT * FROM tokenizer_fast WHERE t &&& 'This is a TEST';