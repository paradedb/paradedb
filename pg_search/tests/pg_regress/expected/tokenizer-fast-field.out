DROP TABLE IF EXISTS tokenizer_fast;
CREATE TABLE tokenizer_fast (
    id serial8 not null primary key,
    t text
);
INSERT INTO tokenizer_fast (t) VALUES ('This is a TEST');
CREATE INDEX idxtokenizer_fast ON tokenizer_fast USING bm25 (id, (t::pdb.literal)) WITH (key_field = 'id');
SELECT * FROM paradedb.schema('idxtokenizer_fast') ORDER BY name;
 name | field_type | stored | indexed | fast | fieldnorms | expand_dots |        tokenizer         | record | normalizer 
------+------------+--------+---------+------+------------+-------------+--------------------------+--------+------------
 ctid | U64        | f      | t       | t    | f          |             |                          |        | 
 id   | I64        | f      | t       | t    | f          |             |                          |        | 
 t    | Str        | f      | t       | t    | f          |             | keyword[lowercase=false] | basic  | raw
(3 rows)

SELECT * FROM tokenizer_fast WHERE t &&& 'This is a TEST';
 id |       t        
----+----------------
  1 | This is a TEST
(1 row)

